packages/smart/contracts/balancer/BPoolForTesting.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+
+import "./BFactory.sol";
+import "../libraries/ERC20.sol";
```

Comment:
This has changed since you started. Now we are use openzepplin's ERC20, from their npm package. Rebase off of dev and there will be examples... it's a small change. As I noted below though, use IERC20 instead of ERC20.